templates/master/01-master-kubelet/_base/units/kubelet.yaml

```diff
@@ -24,6 +24,7 @@ contents: |
         --minimum-container-ttl-duration=6m0s \
         --client-ca-file=/etc/kubernetes/ca.crt \
         --cloud-provider={{cloudProvider .}} \
+        --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \
```

Comment:
This directory doesn't seem to exist on the masters today; is something else going to create it?  Does the kubelet ignore it if it's `ENOENT`?

Reply:
fixed.